ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Node/Serialisation.hs

```diff
@@ -173,6 +183,30 @@ deriving newtype instance
   SerialiseNodeToNode blk (GenTxId blk) =>
   SerialiseNodeToNode blk (WrapGenTxId blk)
 
+-- TODO: move these orphan instances elsewhere
+instance ConvertRawHash blk => SerialiseNodeToNode blk (Point blk) where
+  encodeNodeToNode _ccfg _version = encodePoint $ encodeRawHash (Proxy :: Proxy blk)
```

Comment:
Nitpick: 
```suggestion
  encodeNodeToNode _ccfg _version = encodePoint $ encodeRawHash (Proxy @blk)
```
Also search for "`Proxy`" in the [style guide](https://ouroboros-consensus.cardano.intersectmbo.org/docs/howtos/contributing/style_guide)